### ISSUE TITLE
adds +1 day to release dates from Apple Music

### ISSUE THREAD

**Issue by jgchk** [jgchk/better-rym#35]

**Comment by kopytko95** [comment]
Quick note: It adds a day from every service, not just iTunes

**Comment by jgchk** [comment]
@sc113 @kopytko95 currently the extension grabs release dates in UTC. I believe the issue here is that Apple Music/Spotify/whatever is showing you the release date in your local timezone, so it appears that the date is off.

I can't find any documentation on whether release dates are supposed to be submitted in UTC, but I would assume they should

**Comment by sc113** [comment]
I have no problem with Spotify\other platform dates, the import matches the actual release dates. Also, I'm not sure if the problem with Apple Music is related to the time zone issue.

For example, let's take any release that is tied to a weekly release day (Friday). For example https://en.wikipedia.org/wiki/Scary_Hours_2. Wiki says it was released on March 5. When I import https://open.spotify.com/album/5LuoozUhs2pl3glZeAJl89, it fills correct date - March 5. When I open it on us/gb Apple Music store https://music.apple.com/us/album/scary-hours-2/1556669854, I see the March 5 date everywhere. When I import tags via tag editor scripts from Apple Music (which takes raw api data) - it also returns 2021-03-05T08:00:00Z from every regional store. Judging by the thousand releases for which I've exported tags - every release has T07:00:00Z or T08:00:00Z date (depends on summer time I guess) with no exceptions. So I'm assuming it's a UTC date from Apple Music. But for some reason addon exports 2021-03-06. It may be that Apple Music will return 2021-03-04 for UTC-9 -10 -11 -12 zones (USA west coast/alaska/islands and France islands regions), but for 2021-03-06 there should be UTC+16 zone that doesn't exist: https://en.wikipedia.org/wiki/Time_zone. 
My local time is UTC+3, so I have no problems with Spotify. Judging by @kopytko95's RYM location, he should have UTC+1 and also should not have problems regarding time zones too. Although I'm not sure if Spotify has T08:00:00Z everywhere too, but it probably does.
